java-manta-client-unshaded/src/main/java/com/joyent/manta/config/KeyPairFactory.java

```diff
@@ -53,17 +53,18 @@ public KeyPair createKeyPair() {
         final String keyPath = config.getMantaKeyPath();
         final String password = config.getPassword();
 
+        final char[] charPassword;
+        if (password != null) {
+            charPassword = password.toCharArray();
+        } else {
+            charPassword = null;
+        }
+
```

Comment:
Is this just refactoring this block out of the `try`?

Reply:
The password can now be supplied for both key content and key paths. Passwords were only checked in the `privateKeyContent != null` branch. Notice the usage on [this line](https://github.com/joyent/java-manta/pull/357/files/ed0f307ce37f9fbeebf7c2df94ddce2be67b598f#diff-273d69ef829954db5a617c6424225dd3R67).